curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e9928.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-95
 
 # --description--
 
-I'm not thrilled about that black for the windows anymore. Change the `--window-color1` value to `#bb99ff`.
+With css variables you can change needed values without searching everywhere in the stylesheet. Change the `--window-color1` value to `#bb99ff`.
```

Comment:
```suggestion
With CSS variables you can change values without searching everywhere in the stylesheet. Change the `--window-color1` value to `#bb99ff`.
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix